src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        List<Truck> trucks = new ArrayList<>();
+        for (int i = 0; i < AMOUNT_OF_MODELS; i++) {
```

Comment:
The variable `AMOUNT_OF_MODELS` is not defined within this class or as a static import. Please ensure that this variable is properly declared and initialized. If it is a constant, it should be declared as `private static final int AMOUNT_OF_MODELS = <value>;` at the class level.

Reply:
Yes, it's a constant, i made it in MachineProducer interface.
![image](https://github.com/user-attachments/assets/f2420214-d9e0-4d38-9e1d-04340d92e49e)